gateway/gateway-requests/src/types.rs

```diff
@@ -111,7 +112,12 @@ pub enum ClientControlRequest {
         iv: String,
     },
     #[serde(alias = "handshakePayload")]
-    RegisterHandshakeInitRequest { data: Vec<u8> },
+    RegisterHandshakeInitRequest {
```

Comment:
Have you checked whether the wasm client still works after this renaming?

Reply:
No renaming here, just fmt I think, but I checked the wasm client anyway. It still works